server/svix-server/src/v1/endpoints/event_type.rs

```diff
@@ -280,7 +283,10 @@ async fn get_event_type(
 ) -> Result<Json<EventTypeOut>> {
     let mut query = eventtype::Entity::secure_find_by_name(org_id, evtype_name);
     if let permissions::AllowedFeatureFlags::Some(flags) = feature_flags {
-        query = query.filter(eventtype::Column::FeatureFlag.is_in(flags.into_iter()));
+        let condition = sea_orm::Condition::any()
```

Comment:
Seeing the beefier condition - maybe we could move this as a method on the `eventtype::Entity`? I know we don't do that all the time, although when the sea_orm queries get more verbose I think it's generally cleaner (code reuse arguments aside)

Reply:
I agree.

Just to add another point to this: splitting it to a separate condition variable is unnecessarily verbose and makes things less clear, not more clear.

Reply:
I moved the filtering into an associated function on the entity